libweb3jsonrpc/Net.cpp

```diff
@@ -33,7 +33,23 @@ Net::Net(NetworkFace& _network): m_network(_network) {}
 
 std::string Net::net_version()
 {
-	return "";	// what is this?
+	u256 id = -1;
+	std::string network;
+	dev::WebThreeDirect* web3direct = dynamic_cast<dev::WebThreeDirect*> (&m_network);
```

Comment:
Please don't use dynamic cast. I think it is better to create a new function in NetworkInterface. It does not really fit the topic, but better add a non-fitting function instead of forcing the type to something different here anyway.